browser/brave_profile_prefs.cc

```diff
@@ -269,6 +274,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // change it in upstream.
   registry->SetDefaultPrefValue(prefs::kSafeBrowsingScoutReportingEnabled,
                                 base::Value(false));
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+  brave_vpn::RegisterAndroidProfilePrefs(registry);
```

Comment:
Instead of calling here, you can register android specific prefs at `brave_vpn::RegisterProfilePrefs()`.